Applications/Spire/Source/Ui/NavigationView.cpp

```diff
@@ -131,7 +130,7 @@ NavigationView::NavigationView(std::shared_ptr<CurrentModel> current_model,
   navigation_list_layout->addWidget(m_navigation_list);
   navigation_list_layout->addStretch();
   navigation_menu_layout->addLayout(navigation_list_layout);
-  auto separator = new Separator();
+  auto separator = new Box(nullptr);
```

Comment:
See above, keep the ```Separator``` class and even inherit from ```Box``` to preserve the selector functionality.